server/src/main/java/org/jetbrains/bsp/bazel/server/bsp/managers/BazelBspCompilationManager.java

```diff
@@ -27,7 +27,7 @@ public BepBuildResult buildTargetsWithBep(TargetsSpec targetSpecs, Seq<String> e
             .withFlags(extraFlags.asJava())
             .withTargets(targetSpecs)
             .executeBazelBesCommand()
-            .waitAndGetResult();
+            .waitAndGetResult(false);
```

Comment:
do we want to wait for the entire output for run & test in the `ExecuteService`?

Reply:
sorry I'm not sure what you're asking exactly here?

Reply:
we are calling bazel in `ExecuteService` as well - now with flag set on true, and the question is do we want to use the sync collector there or maybe the async one?

Reply:
ah, sorry I misread that as `ExecutorService` and got very confused :D